docs/04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Enable Kyma with JetStream
+---
+
+This guide shows how to enable JetStream and how it upholds `at least once delivery` quality of service.
+
+### Enable JetStream
+
+Install Kyma and enable the JetStream flag by running:
+
+```
+kyma deploy --value global.jetstream.enabled=true --value global.jetstream.storage=file
+```
+
+> **NOTE:** The storage flag set to `file` enables persistence of messages and streams, in case of NATS server restart.
+
+### At least once delivery guarantees using JetStream backend
```

Comment:
```suggestion
### `At least once` delivery guaranteed using JetStream backend
```

Reply:
How about I remove the word `guaranteed` itself?

Reply:
And do I need the highlight for the heading?